.github/workflows/e2e-tests.yml

```diff
@@ -81,9 +81,15 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
-          go-version: '1.21' 
+          go-version: '1.21'
           check-latest: true
 
+      - uses: imjasonh/setup-crane@00c9e93efa4e1138c9a7a5c594acd6c75a2fbf0c # v0.3
```

Comment:
> Is this step necessary for the additional e2e test below?

Yes, `crane` is  listed among the requirements for the new e2e test - https://github.com/sigstore/cosign/pull/3464/files#diff-605c1fbbe4548795bd84031f2d7973b208f83528d765fdd07f8d4891d38bd6e3R25, and in the test/e2e_tsa_certbundle.sh test, `crane` called [her](https://github.com/sigstore/cosign/pull/3464/files#diff-605c1fbbe4548795bd84031f2d7973b208f83528d765fdd07f8d4891d38bd6e3R40) and [here](https://github.com/sigstore/cosign/pull/3464/files#diff-605c1fbbe4548795bd84031f2d7973b208f83528d765fdd07f8d4891d38bd6e3R43).

[crane](https://pkg.go.dev/github.com/google/go-containerregistry/cmd/crane) - `a tool for interacting with remote images and registries` (I think of it as a 'docker light' for creating or publishing ISO images) - is used in several functional tests under the [test](https://github.com/sigstore/cosign/tree/main/test) directory ([grep hits](https://gist.github.com/dmitris/4475bf11057260a70dac0472e4b25b37)).  In the same file, that `setup-crane` is already used twice - https://github.com/sigstore/cosign/blob/main/.github/workflows/e2e-tests.yml#L48 and https://github.com/sigstore/cosign/blob/main/.github/workflows/e2e-tests.yml#L67 without additional comments, so  I wonder if the comment would break the consistency? There are no comments for other `uses` lines either so personally, I would prefer not to add it to avoid "visual clutter" 😄 (since the purpose of it would be clear and intuitive to most people working with the functional tests and their configuration).